requirements/dev.in

```diff
@@ -11,7 +11,7 @@ PyPOM
 Django==3.0.8  # match common.in for faster install
 
 # for git commit hooks
-pre-commit==2.2.0  # prettier requires at least version 10.13.0 of Node, please upgrade
+pre-commit
```

Comment:
Now that we do not run `pre-commit`, we can run its replacements in the task with Node